tests/cisco_ios/show_ip_bgp_neighbors/cisco_ios_show_ip_bgp_neighbors_multiple.raw

```diff
@@ -0,0 +1,1239 @@
+
+=============================================================================
+Warning!
+"This system is for the use of authorized users only.  Individuals using this
+computer system without authority, or in excess of their authority, are
+subject to having all of their activities on this system monitored and
+recorded by systems personnel.  In the course of monitoring individuals
+improperly using this system, or in the course of system maintenance, the
+activities of authorized users will also be monitored."
+-----------------------------------------------------------------------------
+HMH
+nsxe-core-rtr2
+=============================================================================
+BGP neighbor is 175.135.172.146,  remote AS 65180, internal link
+ Description: nsxe-core-rtr1
+ Member of peer-group HMCO-core for session parameters
```

Comment:
I would update any values that are using **\w+** and change them to **\S+**. This will capture the `peer_group: 'HMCO-core'` within the parsed file.